akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -381,6 +408,11 @@ sealed abstract class ByteString extends IndexedSeq[Byte] with IndexedSeqOptimiz
   def asByteBuffers: immutable.Iterable[ByteBuffer]
 
   /**
+   * Indicates whether this ByteString can produce a wrapped ByteBuffer via [[asByteBuffer]].
```

Comment:
scaladoc error? seeings lots of these in the build tho 

[warn] /mnt/akka/jenkinsakka/localhome-a0/workspace/pr-validator-per-commit-jenkins/akka-actor/target/java/akka/util/ByteString.java:207: warning - Tag @link: reference not found: asByteBuffer